config/initializers/flicker.rb

```diff
@@ -0,0 +1,2 @@
+ENV["FLICKR_KEY"] = 'b2aef871a11feaaacea0b0095199ff81'
```

Comment:
Please 'git reset --soft HEAD~1' to undo this commit, remove the API keys, recommit and repush. We do not want this visible on the repo.